### ISSUE TITLE
Showing tooltip value for stacked column chart

### ISSUE THREAD

**Issue by lawrenceryan** [amcharts/amcharts4#4282]
hi, i'm using amcharts4 and trying to show tooltip with grouped value from stacked column chart based on cursor position, but the chart value doesn't appear in the tooltip just like in the screen shot

<img width="525" alt="Screen Shot 2023-06-06 at 13 58 25" src="https://github.com/amcharts/amcharts4/assets/52479198/94888269-1492-4a56-9202-6419cb62ac63">

here is the snippet of the code 

```
chart.cursor.events.on('cursorpositionchanged', function(ev) {
  series.adapter.add('tooltipText', function(text) {
    var text = '[bold]{dateX}[/]\n'
    chart.series.each(function(item) {
      if (!item.isHiding) {
        text += `[${item.stroke.hex}]●[/] ${item.name} : {${item.dataFields.valueY}}\n`
      }
    })
    chart.tooltipText = text
  })
})
```

what can i do to show the chart value?

**Comment by martynasma** [comment]
You're adding a new adapter every time cursor moves. That does not seem right.

You don't need the "cursorpositionchanged" event at all.

You can just add `tooltipText` adapter to series. The cursor will use it automatically when moved.

Example:
https://www.amcharts.com/docs/v4/tutorials/multi-series-shared-tooltip-with-colored-bullets/

**Comment by lawrenceryan** [comment]
Thank you for the follow up!

I tried to follow the example but instead i got this, i only want the tooltip to show once instead of in every series and i followed the example.

Is it maybe because i put it in the series instead of the chart?

<img width="567" alt="Screen Shot 2023-06-07 at 12 39 08" src="https://github.com/amcharts/amcharts4/assets/52479198/cfc7e385-71d3-4452-a61f-29a56a4fb391">

**Comment by martynasma** [comment]
You can use `maxTooltipDistance` to limit number of tooltips:

https://www.amcharts.com/docs/v4/concepts/chart-cursor/#Limiting_number_of_series_tooltips

**Comment by lawrenceryan** [comment]
Ah, i see. it will only show 1 tooltip if I set to -1, but will still showing the other tooltip if i hover the bar, is that expected?

**Comment by martynasma** [comment]
No, it's not expected, unless you added `tooltipText` to the column template, too. It should be just on series.

**Comment by lawrenceryan** [comment]
I re-checked the code and I think i only set it on the series.

But according to this example, I tried setting `maxTooltipDistance` to -1, it only shows 1 at a time but if i hover to other line, it still showing too.

[https://www.amcharts.com/docs/v4/concepts/chart-cursor/#Limiting_number_of_series_tooltips](url)

I'm just wondering, is is possible to show the tooltip based on the cursor position at this point?

**Comment by martynasma** [comment]
Would you be able to post your whole chart and data on CodePen/jsFiddle so that we can test?

**Comment by lawrenceryan** [comment]
Okay, I'll come back later after i made it on CodePen/jsFiddle.

**Comment by lawrenceryan** [comment]
Here is the whole chart code with example data of my current situation

[https://codesandbox.io/s/stacked-column-chart-with-tooltip-t2wjvy?file=/src/App.js](url)

**Comment by lawrenceryan** [comment]
Oh, I'm sorry, I've update the code, can you please check it one more time?

[https://codesandbox.io/s/stacked-column-chart-with-tooltip-t2wjvy](https://codesandbox.io/s/stacked-column-chart-with-tooltip-t2wjvy)

**Comment by martynasma** [comment]
Thanks. It's working now.

Though, I can't make it show two tooltips at once. It's displaying one tooltip at a time. Or do you want it to always show in the same place at the top of the stack?

In such case, you'd need to add another series with zero values and add `tooltipText` only to it.

https://codesandbox.io/s/stacked-column-chart-with-tooltip-forked-580d2n

**Comment by lawrenceryan** [comment]
Oh, this is the one that I wanted! Thank you so much for helping!

**Comment by lawrenceryan** [comment]
Regarding the code above, I added tooltip to the scrollbarx.
When I hover it for the first time, the tooltip was showing, but the next time I hover it, only the chart tooltip was showing.

The code that I added was
chart.scrollbarX.tooltipText = 'This is scrollbar'

How can I fix this so when i hover the scrollbar, the chart tooltip wont showing, and when i hover the chart, only the chart tooltip will show? I'm using amcharts 4.9.37
